dashboard/views.py

```diff
@@ -119,7 +117,7 @@ def file_access_within_week(request, course_id=0):
         "grade": grade,
         "course_id": course_id
     }
-    eventlog(request.user, EVENT_VIEW_FILE_ACCESS, extra=data)
+    eventlog(request.user, EventLogTypes.EVENT_VIEW_FILE_ACCESS.value, extra=data)
```

Comment:
I think EventLogTypes.EVENT_VIEW_FILE_ACCESS should be good enough, without use ".value"

Reply:
as per the suggestion, it will load the event as `EventLogTypes.EVENT_VIEW_FILE_ACCESS` instead of `VIEW_SET_DEFAULT` or `VIEW_FILE_ACCESS`